### PR TITLE
Update maven-wrapper to 3.3.0 & dependencies

### PR DESCRIPTION
**Pull request by wendigo**
Wrapper regenerated using: ./mwnv wrapper:wrapper -Dtype=script-only

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -277,7 +277,7 @@
             <dependency>
                 <groupId>org.apache.arrow</groupId>
                 <artifactId>arrow-bom</artifactId>
-                <version>15.0.2</version>
+                <version>16.0.0</version>
```
Comment (losipiuk) [inline pom.xml:280]: let's wait for 16.0.1

Comment (wendigo) [reply pom.xml:280]: Why?

Comment (losipiuk) [reply pom.xml:280]: 0.0 is probably shit

Comment (wendigo) [reply pom.xml:280]: Doubt that, given the fact that we test arrow in BigQuery connector :) The 16.0.0 doesn't mean that they've rewrote or change the API of the java library but merely that they've added new features that were not there before

**Review by losipiuk — APPROVED** [review]

**Pull request merged**